src/Components/Open/OpenModelControl.test.jsx

```diff
@@ -14,8 +14,10 @@ describe('Open Model Dialog', () => {
     const {getByTestId, getByText} = render(<OpenModelControlFixture/>)
     const openControlButton = getByTestId('control-button-open')
     fireEvent.click(openControlButton)
+    const GithubTab = getByText('Github')
+    fireEvent.click(GithubTab)
     const loginTextMatcher = (content, node) => {
-      const hasText = (_node) => _node.textContent.includes('Please login to GitHub')
+      const hasText = (_node) => _node.textContent.includes('Host your IFC models on GitHub and log in to Share')
```

Comment:
Let's not use IFC anymore, to prepare for multi-format.  Just "Host your models on ..."